tools/model-compiler/src/main/java/io/spine/tools/gradle/compiler/GeneratedInterfaces.java

```diff
@@ -155,6 +164,10 @@ public PostfixPattern endsWith(String postfix) {
         return new PostfixPattern(postfix);
     }
 
+    private PostfixPattern endsWith(MessageFile messageFile) {
```

Comment:
Before merging let's discuss why you need this method in scope of this changeset.